src/messages/manifestjson.js

```diff
@@ -106,6 +106,20 @@ export function manifestPropMissing(property) {
   };
 }
 
+export const MANIFEST_ICON_NOT_FOUND = 'MANIFEST_ICON_NOT_FOUND';
+export function manifestIconMissing(path) {
+  return {
+    code: MANIFEST_ICON_NOT_FOUND,
+    legacyCode: null,
+    message: _(
+      'An icon defined in the manifest could not be found in the package.'),
+    description: sprintf(
+      _('Icon could not be found at "%(path)s".'),
+      {path}),
```

Comment:
I like this I think. We should apply this to how `UNSUPPORTED_API` is defined too.